crates/matrix-sdk/src/client/mod.rs

```diff
@@ -871,6 +875,18 @@ impl Client {
     ///     // Use the context
     /// });
     ///
+    /// // This will handle membership events in joined rooms. Invites are special, cf below.
```

Comment:
You'd need a dot for the `cf.`, but let's just change it to something simpler.

```suggestion
    /// // This will handle membership events in joined rooms. Invites are special, see below.
```